src/Control/Lens/Fold.hs

```diff
@@ -361,6 +362,24 @@ filtered :: (Choice p, Applicative f) => (a -> Bool) -> Optic' p f a a
 filtered p = dimap (\x -> if p x then Right x else Left x) (either pure id) . right'
 {-# INLINE filtered #-}
 
+-- | Obtain a potentially empty 'IndexedFold' by taking the first element from another,
```

Comment:
should it mention `IndexedTraversal`, as the more concrete type signature uses it?